src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,40 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    public static final int WORD_PART = 0;
+    public static final int NUMBER_PART = 1;
```

Comment:
```suggestion
    private static final int NUMBER_PART = 1;
    private static final String CSV_SEPARATOR = ",";
```
if we don't use constants outside let's make them private